content/vendor/dashboard/application-status.md

```diff
@@ -11,7 +11,7 @@ With minimal additional to the kots.io Application YAML, it is possible to displ
 
 It is necessary to target specific Kubernetes resources for the dashboard to accurately report status. We suggest at least one resource be added. Resources that are currently supported are Deployments, StatefulSets, Services, Ingresses and PersistentVolumeClaims.
 
-## Kots Application Spec
+{{< linked_headline "Kots Application Spec" >}}
```

Comment:
I don’t understand this. Any reason not to use standard Markdown here? the `linked_headline` is proprietary compared to the ## syntax.

Reply:
You are right, this is not necessary.  I had copied this from help-center, as an expedient for making hover-over anchor links, but this is not necessary, and Hugo will still make anchor links from regular markdown headers.

Removing